packages/ui/src/components/NavBar.vue

```diff
@@ -18,6 +18,13 @@
           <router-link class="tab" to="/interactions">
             Scans history
           </router-link>
+          <a
+            class="tab"
+            href="https://witnet.network/search/b9d56f8e178fab41f0d497ac3d2cd6c943d33977f92ca8adfb602512a4eb6975"
+            target="_blank"
+          >
+            Witnet audit
```

Comment:
I find "Witnet audit" abit misleading, most people would think it's a security audit. What do you think about something like "Witnet proof"?